### ISSUE TITLE
It is possible to run the slider in a "vertical" mode?

### ISSUE THREAD

**Issue by GiancarlosIO** [NickPiscitelli/Glider.js#47]

**Comment by NickPiscitelli** [comment]
Sorry for the late reply, unfortunately vertical mode is not yet supported,
but I've marked it as a feature request for future releases.

**Comment by GiancarlosIO** [comment]
@NickPiscitelli oh, I see. I will be waiting for this feature :)
Thank you!

**Comment by kwdowik** [comment]
@NickPiscitelli I can try to implement this feature. Do you mind?
Cheers

**Comment by NickPiscitelli** [comment]
Not at all, would love to accept a pull request. I imagine the hardest part
about this would be swapping between horizontal and vertical on breakpoint
change.

**Comment by filip-jk** [comment]
Hi there!

Is there any progress with this feature/enhancement? We would love to use Glider.js in our upcoming projects but this is a deal-breaker. :( 

Thanks!

**Comment by NickPiscitelli** [comment]
There are no upcoming releases.
